tests/functional/u/unused/unused_private_member.py

```diff
@@ -135,3 +135,44 @@ def __init__(self):
     def attr_c(self):
         """Get c."""
         return cls.__attr_c  # [undefined-variable]
+
+# Test cases for false-positive reported in #4673
```

Comment:
Can you add an additional empty line here?

Reply:
This is weird, I've added one but it doesn't seem to show up in the ChangeLog

Reply:
```suggestion

# Test cases for false-positive reported in #4673
```

Reply:
Ah okay, seems okay now if I look at the files changed view

Reply:
Sometimes empty lines get added to the end of a section in diff view